backend/database/MatchHistorySeeder.json

```diff
@@ -0,0 +1,364 @@
+{
+  "MatchHistory": [
+    {
+      "PutRequest": {
+        "Item": {
+          "user1ID": {
+            "S": "1"
+          },
+          "date": {
+            "S": "20210101"
```

Comment:
We should store the dates in the ISO8601 format across our whole application. As such, the date string will look something like this:
```
2022-10-03T00:55:40.089Z
``` 

If we standardise it, the way we parse dates, whether that be for displaying match history on the frontend or getting matches in our connecting algorithm will all be the same. ISO8601 is probably the best because it integrates really well with Javascript/Typescript. E.g. this is how you get the current ISO timestamp in JS:
```
const date = new Date();
const isoDateString = date.toISOString();
```